doc/man3/X509_PUBKEY_new.pod

```diff
@@ -85,13 +88,17 @@ d2i_PUBKEY_bio(), d2i_PUBKEY_fp(), i2d_PUBKEY_bio() and i2d_PUBKEY_fp() are
 similar to d2i_PUBKEY() and i2d_PUBKEY() except they decode or encode using a
 B<BIO> or B<FILE> pointer.
 
+X509_PUBKEY_set0_public_key() sets the public key encoding of I<pub>
+to the I<penclen> bytes contained in buffer I<penc>.
+I<penc> may be NULL to indicate that there is no actual public key data.
+
 X509_PUBKEY_set0_param() sets the public key parameters of I<pub>. The
 OID associated with the algorithm is set to I<aobj>. The type of the
 algorithm parameters is set to I<type> using the structure I<pval>.
 The encoding of the public key itself is set to the I<penclen>
-bytes contained in buffer I<penc>. On success ownership of all the supplied
-parameters is passed to I<pub> so they must not be freed after the
-call.
+bytes contained in buffer I<penc>, which may not be NULL.
```

Comment:
This is incorrect according to the implementation, isn't it? If penc == NULL it is just ignored and not set. I.E. rather say something like: `If I<penc> is not NULL the encoding of the public key itself is set to the I<penclen> bytes contained in buffer I<penc>.`

Reply:
Ah, right! Changed this way.
On this occasion further improved the doc of both `X509_PUBKEY_set0_param()` and of the new `X509_PUBKEY_set0_public_key()`.